.gulp/common.iced

```diff
@@ -384,7 +384,7 @@ module.exports.task 'release-only', '', (done)->
   done()
  
 configString = (s)->
-  "#{s.charAt 0 .toUpperCase()}#{s.slice 1 .toLowerCase() }"
+  "#{(s.charAt 0).toUpperCase()}#{(s.slice 1).toLowerCase() }"

```

Comment:
@fearthecowboy FYI. This translated to
``` JavaScript
return "" + (s.charAt(0..toUpperCase())) + (s.slice(1..toLowerCase()));
```
which yields a
> TypeError: 0.toUpperCase is not a function

Did iced coffee script change recently? 😮